src/text-encoder.js

```diff
@@ -1,5 +1,4 @@
 'use strict'
 
-const { TextEncoder } = require('web-encoding')
-
-module.exports = TextEncoder
+// This is fine for React Native as long as the environment is polyfilled with text-encoding
+module.exports = require('web-encoding').TextEncoder
```

Comment:
can you check this PR https://github.com/Gozala/web-encoding/pull/13 and see if its enough ?

Reply:
Yep, it should work. I'll have a look. Thanks!

Reply:
It seems to be working fine! I'm updating `web-encoding` to the latest version.

Reply:
I've just removed the comment because it is no longer necessary.